tools/gdb/README.md

```diff
@@ -0,0 +1,41 @@
+# GDB image to load Agent core dump
+
+Assuming you have a core dump generated by a containerized Agent (with
+the `c_core_dump` Agent option enabled), load the core dump with GDB with the provided Dockerfile.
+
+The image built from the Dockerfile includes:
+
+* the debug symbols of the Agent (datadog-agent-dbg), for the version you're debugging
+* the GDB scripts for Go and Python (only tested with Python 3)
+
+The image is based on the `datadog/agent` image for the version of the Agent you're debugging, so that the debug symbols of
```

Comment:
```suggestion
The image is based on the `datadog/agent` image for the version of the Agent you're debugging, so that the debug symbols of the system libraries are at the correct version.
```